src/language-js/print/binaryish.js

```diff
@@ -110,7 +111,8 @@ function printBinaryishExpression(path, options, print) {
       grandparent.type !== "ReturnStatement" &&
       grandparent.type !== "ThrowStatement" &&
       !isCallExpression(grandparent)) ||
-    parent.type === "TemplateLiteral";
+    parent.type === "TemplateLiteral" ||
+    (isCallExpression(parent) && getCallArguments(parent).length === 1);
```

Comment:
The call expression have different children, we need make sure it's argument, not callee.